pages/content/posts/swag-at-tpac-anaheim.md

```diff
@@ -0,0 +1,38 @@
+---
+author: "Will Bamberg"
+title: "Talking about security docs at TPAC Anaheim"
+date: "2024-10-18"
+tags: ["announcements"]
+ShowToc: false
+ShowBreadCrumbs: false
+excerpt: "At TPAC Anaheim, we talked with security experts about security documentation needs."
+---
+
+In September, Open Web Docs attended [TPAC](https://www.w3.org/2024/09/TPAC/), the main annual meeting for the W3C. It's the first time we've been together for almost 2 years, and it was great to reconnect with each other and with many of our collaborators. We proposed and chaired a [breakout session](https://github.com/w3c/tpac2024-breakouts/issues/96) about the activities of the [Security Web Application Guidelines Community Group (SWAG CG)](https://www.w3.org/community/swag/), and in this post we'd like to summarise some of the discussions that came out of this.
+
+The SWAG CG was formed by [Dan Appelquist](https://mastodon.social/@torgo) in June 2024 and has been meeting weekly since then. Its mission is:
+
+> To increase the overall security of web application development, thereby making the web a more secure platform for web users. It will accomplish this by writing security best practices for web developers and providing a platform for stakeholder collaboration.
+
+Open Web Docs participates in this group because we think better documentation can improve the state of web security, and we think the SWAG CG can reveal gaps in web security documentation and provide technical support for us as we work to fill them.
```

Comment:
```suggestion
Open Web Docs participates in this group because we think better documentation will improve the state of web security, and we think the SWAG CG can reveal gaps in web security documentation and provide technical support for us as we work to fill them.
```